tools/mkdfs/mkdfs.c

```diff
@@ -288,7 +412,8 @@ int main(int argc, char *argv[])
 
         return -1;
     }
-
+    sort_dfs_entries();
+    write_dfs_entries();
```

Comment:
I would use the name "dfs lookup" here too. So maybe just do one function write_dfs_lookup() (which will also take care of sorting)